clusters/threat-actor.json

```diff
@@ -2689,7 +2689,6 @@
           "https://dragos.com/blog/crashoverride/CrashOverride-01.pdf",
           "https://www.us-cert.gov/ncas/alerts/TA17-163A",
           "https://ics.sans.org/blog/2016/01/09/confirmation-of-a-coordinated-attack-on-the-ukrainian-power-grid",
-          "https://www.cfr.org/interactive/cyber-operations/black-energy",
```

Comment:
Dead link

Reply:
Thanks. It reminds me that I should pass all the links from the galaxies to webarchive.org…